include/seqan3/search/dream_index/interleaved_bloom_filter.hpp

```diff
@@ -386,9 +391,14 @@ class interleaved_bloom_filter
     {
         assert(result_buffer.size() == bin_count());
 
-        std::vector<size_t> idx = hash_seeds;
+        std::array<size_t, 5> idx;
```

Comment:
```suggestion
        std::array<size_t, 5> bloom_filter_indices; // which bits to check in each individual bloom filter
```
? `idx` does not tell me anything